Cloud Application Manager/Deploying Anywhere/using-centurylink-dec3.md

```diff
@@ -0,0 +1,122 @@
+* [Deploy to CenturyLink DEC3 from Cloud Application Manager](#deploy-to-centurylink-dec3-from-cloud-application-manager)
+* [Naming vCloud resources ???](#naming-vcloud-resources-)
+* [Discovering and Registering Existing instances](#discovering-and-registering-existing-instances)
+* [Contacting Cloud Application Manager Support](#contacting-cloud-application-manager-support)
+
+### Overview
+
+This article is meant to assist users of Cloud Application Manager to learn how to deploy a CenturyLink DEC 3 Provider, available only in Latin America region.
+
+### Audience
+
+All Cloud Application Manager users who wants to deploy workloads into a CenturyLink DEC 3 Provider.
+
+### Prerequisites
+
+* You must have a Cloud Application Manager Account
+* Your base URL, and user credentials for your CenturyLink DEC 3 subscription
+
+### Connect CenturyLink DEC3 environment in Cloud Application Manager
+
+Follow these steps to add a provider in Cloud Application Manager that integrates with your CenturyLink DEC 3 environment. Each provider connects to the virtual data center in that specific location.
+
+#### Add a provider in CenturyLink DEC3
+
+1. In Cloud Application Manager, on the Providers page, click **New Provider**.
+
+2. Select **CenturyLink DEC3** and fill in the required fields:
+
+    ![New DEC3 provider](../../images/cloud-application-manager/deploying-anywhere/dec3/new-dec3-provider.png)
+
+   * **URL**. Enter the URL to access the CenturyLink DEC3 portal
+   * **Tenant**. Select the tenant your environment is hosted into.
+   * **Domain**. Enter the domain of your environment
+   * **Username**. Enter the username to log in to your CenturyLink DEC3 account.
+   * **Password**. Enter the password for your CenturyLink DEC3 account.
+
+### Deploy to CenturyLink DEC3 from Cloud Application Manager
+
+Create a [deployment policy](../Automating Deployments/deploymentpolicy-box.md) to encapsulate parameters required to deploy Linux or Windows instances through the CenturyLink DEC3 provider
+
+![New CenturyLink DEC3 Policy Box](../../images/cloud-application-manager/deploying-anywhere/dec3/new-dec3-deployment-policy.png)
+
+At deploy time, Cloud Application Manager auto provisions a vRealize instance and orchestrates workload automation on them.
+
+**Note:** Auto provisioning may fail if you specify virtual CPUs, memory, and virtual disk capacity that exceeds the limit in your virtual data center.
+
+Edit the deployment policy to set the following parameters for deploying to CenturyLink DEC3.
+
+#### Resource
+
+![Resource section of DEC3 deployment policy](../../images/cloud-application-manager/deploying-anywhere/dec3/dec3-dp-resource.png)
+
+| Option | Description |
+|--------|-------------|
+| Catalog Item | Select the template from the catalog to be used for deployment. VMs imported as OVFs then uploaded as templates are also listed. |
```

Comment:
Please, remove this: `VMs imported as OVFs then uploaded as templates are also listed`